docs/components/fab.md

```diff
@@ -0,0 +1,369 @@
+
+For more information on getting started with the Material for Flutter, go to the Flutter [Material library](https://api.flutter.dev/flutter/material/material-library.html) page.
+
+
+### Making FABs accessible
+
+Flutter's APIs support accessibility setting for large fonts, screen readers, and sufficient contrast. For more information, go to Flutter's [accessibility](https://flutter.dev/docs/development/accessibility-and-localization/accessibility) and [internationalization](https://flutter.dev/docs/development/accessibility-and-localization/internationalization) pages.
+
+For more guidance on writing labels, go to [our page on how to write a good accessibility label](https://material.io/design/usability/accessibility.html#writing).
+
+
+
+## Regular FABs
+
+
+Regular FABs are FABs that are not expanded and are a regular size.
+
+`FloatingActionButton`
+* [Class description](https://api.flutter.dev/flutter/material/FloatingActionButton-class.html)
+* [GitHub source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/floating_action_button.dart)
+* [Dartpad Demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/fabs/regular&gh_ref=develop)
+
+
+
+The following example shows a regular FAB.
+
+!["Regular FAB example for Flutter with white '+' icon over a black background."](assets/fabs/regular_fab.png)
+
+```dart
+FloatingActionButton(
+  backgroundColor: Colors.black,
+  foregroundColor: Colors.white,
+  onPressed: () {
+    // Respond to button press
+  },
+  child: Icon(Icons.add),
+)
+```
+
+### Anatomy and key properties
+
+![Regular FAB anatomy diagram](assets/FAB_anatomy-long.png)
+
+A regular FAB has a container and an icon.
+
+1. Container
+1. Icon
+
+#### Container attributes
+
+&nbsp;         | Property
+-------------- | -----------------------
+**Color** | `backgroundColor`
+**Stroke color** | `color` of `side` on a custom `shape`
+**Stroke width** | `width` of `side` on a custom `shape`
+**Shape** | `shape`
+**Elevation** | `elevation`
+**Ripple color** | `splashColor`
+
+#### Icon attributes
+
+&nbsp;         | Property
+-------------- | ------------------------
+**Icon** | `child`
+**Color** | `foregroundColor`
+**Size** | `size` on `child` when using an `Icon`
+
+## Mini FABs
+
+
+A mini FAB should be used on smaller screens.
+
+Mini FABs can also be used to create visual continuity with other screen elements.
+
+### Mini FABs example
+
+`FloatingActionButton`
+* [Class description](https://api.flutter.dev/flutter/material/FloatingActionButton-class.html)
+* [GitHub source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/floating_action_button.dart)
+* [Dartpad Demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/fabs/mini&gh_ref=develop)
```

Comment:
There doesn't appear to be any gist source code.

Reply:
The dartpad demos are part of this PR, so these links will begin working after I merge it.